src/saleor_app/webhook.py

```diff
@@ -0,0 +1,80 @@
+from typing import Callable, List
+
+from fastapi import APIRouter, Depends, Header, HTTPException, Request
+from fastapi.routing import APIRoute
+from starlette.responses import Response
+
+from saleor_app.deps import (
+    saleor_domain_header,
+    verify_saleor_domain,
+    verify_webhook_signature,
+)
+from saleor_app.schemas.handlers import (
+    SaleorEventType,
+    SQSHandler,
+    SQSUrl,
+    WebHookHandlerSignature,
+)
+from saleor_app.schemas.webhook import Webhook
+
+SALEOR_EVENT_HEADER = "x-saleor-event"
+
+
+class WebhookRoute(APIRoute):
```

Comment:
I don't think that this is used anywhere. Do we need this?

Reply:
`WebhookRoute` is an important element of `WebhookRouter` in `SaleorApp.include_webhook_router`